libs/ledger-live-common/src/exchange/swap/getProviders.ts

```diff
@@ -2,34 +2,21 @@ import qs from "qs";
 import { getEnv } from "../../env";
 import { SwapNoAvailableProviders } from "../../errors";
 import network from "../../network";
-import {
-  getAvailableProviders,
-  getSwapAPIBaseURL,
-  getSwapAPIVersion,
-} from "./";
+import { getAvailableProviders, getSwapAPIBaseURL } from "./";
 import { mockGetProviders } from "./mock";
 import type { GetProviders, ProvidersResponseV4 } from "./types";
 
 const getProviders: GetProviders = async () => {
   if (getEnv("MOCK")) return mockGetProviders();
 
-  const version = getSwapAPIVersion();
-
   const res = await network({
     method: "GET",
     url: `${getSwapAPIBaseURL()}/providers`,
-    params: version >= 3 ? { whitelist: getAvailableProviders() } : undefined,
+    params: { whitelist: getAvailableProviders() },
```

Comment:
won't `getAvailableProviders` currently just return the CEX providers? Do you have anything in mind with how we'd add the DEX providers here too?

Reply:
@cjordan-ledger `getAvailableProviders` is the list of providers supported by frontend, it is hardcoded. So they will be added in the PR that adds support for paraswap and 1inch I guess. Then the backend only inludes in its response providers that are in this whitelist. This mechanism is to manage backward compatibility / different release cycles between backend and frontend (for instance, current prod backend already supports paraswap and 1inch).